README.md

```diff
@@ -1 +1,78 @@
-# portfoliopage
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> It is portfolio website in the making !!
+
+
+## Built With
+
+- html, css, js
+- Linters
+
+## Live Demo (if available)
+
+[coming soon](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+ - vs code
+ - Git
+
+### Setup
+1. Download the Zip file or clone the repo with:
+`git clone https://github.com/belmeetmule/portfolio.git`
+
+### Install
+> To install linters and other project's dependencies run:
+` npm install `
+
+
+### Run tests
+For tracking linter errors locally you need to follow these steps:
+
+  > 1. For tracking linter errors in HTML files run:
+  ` npx hint . `
+
+  >2. For tracking linter errors in CSS or SASS files run:
+  ` npx stylelint "\*_/_.{css,scss}" `
+
+
+## Authors
+
+👤 **Habtamu Birru**
+
+- GitHub: Habtamu2010
+- Twitter: beleyaha
+- LinkedIn: habtamu-birru-4187ab20
```

Comment:
- [ ] Kindly make sure to update the working links of your social media and github profile. It will make your README more professional. :wink: